upa/src/tool/submitProofs.ts

```diff
@@ -55,7 +54,6 @@ export const submitProofs = command({
     isDryRun: flag({
       type: boolean,
       long: "dry-run",
-      short: "n",
```

Comment:
This one is quite standard.  I'd be keep to keep it and remove or update the one above.

Reply:
Ok, removed the `numProofs` one